queries/windows_filespace.kusto

```diff
@@ -9,20 +9,22 @@ let Disks = dynamic(["C:"]);
 let warning = int(10);
 let critical = int(5);
 let appl_groups = ("RG.*AVD*");  // excluding a list of ResourceGroups
-Perf
+InsightsMetrics
 | where TimeGenerated > ago(5m)
-| where ObjectName contains "LogicalDisk"
-| where CounterName == "% Free Space"
-| where InstanceName in (Disks)
-| where CounterValue <= (warning)
-| extend state = case(CounterValue >= (critical), "Warning", CounterValue <= (critical), "Critical", "Warning")
-| extend threshold = case(CounterValue >= (critical), warning, CounterValue <= (critical), critical, warning)
+| where Namespace == "LogicalDisk"
+| where Name == "FreeSpacePercentage"
+| extend Mountpoint = tostring(parse_json(Tags)["vm.azm.ms/mountId"])
+| where Mountpoint in (Disks)
+| extend InverseVal = (100 - Val)
+| where InverseVal <= (warning)
+| extend state = case(InverseVal >= (critical), "Warning", InverseVal <= (critical), "Critical", "Warning")
+| extend threshold = case(InverseVal >= (critical), warning, InverseVal <= (critical), critical, warning)
```

Comment:
This switch case seems to be the wrong way around. It currently says "if value larger that critical it's a warning", right?